hknweb/events/tests/views/event_transactions/test_repeat_event.py

```diff
@@ -0,0 +1,87 @@
+from datetime import datetime, timedelta
+from django.test import TestCase
+
+from django.urls import reverse
+
+from hknweb.events.tests.views.event_transactions.utils import setUp
+from hknweb.events.utils import generate_recurrence_times
+
+class RepeatIntervalsUtilsTests(TestCase):
+    def test_repeat_time_intervals(self):
+        start_time = datetime(2022, 3, 15, 11, 30)
+        end_time = datetime(2022, 3, 15, 3, 30)
+
+        def base_tests():
```

Comment:
What's the reasoning behind wrapping this in base_tests?

Reply:
Nothing special, just grouping them as a name of a series of small base case sanity checks

Reply:
Why not just make them a test case or unwrap them?

Reply:
Agreed on making it a separate one, just did it